metis-normalization/src/main/java/eu/europeana/normalization/dates/edtf/EdtfParser.java

```diff
@@ -12,15 +12,12 @@
  */
 public class EdtfParser {
 
-
+  // TODO: 21/12/2022 This is used transparently for both EDTF parsing as well as Dcmi parsing which is probably
+  //  incorrect, because of the allowance of the [?%~] modifiers.
```

Comment:
the modifiers `~%?` are part of level 1 for EDTF
https://www.loc.gov/standards/datetime/
maybe add a reference to this site as a follow up

Reply:
This is part of the documentation that Nuno has provided.
I've added it in the comment nonetheless.